pioneer/packages/joy-utils/src/react/context/transport.tsx

```diff
@@ -2,7 +2,7 @@ import React, { createContext, useContext } from "react";
 import { ApiContext } from "@polkadot/react-api";
 import { ApiProps } from "@polkadot/react-api/types";
 
-import Transport from "../../transport";
+import { Transport } from "../../transport";
```

Comment:
I can clearly see by examining `../../transport/index.ts` that there is a default export class statement which means this fix is incorrect, but I can't figure out how my IDE is not seeing the same thing :)

<img width="1046" alt="Screen Shot 2020-05-29 at 9 19 48 PM" src="https://user-images.githubusercontent.com/1621012/83292055-23066280-a1f2-11ea-92e8-ca1176321174.png">

Reply:
I think I know what's the issue here. There is `joy-utils/src/Transport` (file) and `joy-utils/src/transport` (directory). Perhaps in some environments where paths are considered case insensitive import from file takes priority over the import from directory and hence the error.